Utilities/Log.cpp

```diff
@@ -393,7 +394,12 @@ logs::file_listener::file_listener(const std::string& name)
 	ver.m.ch  = nullptr;
 	ver.m.sev = level::always;
 	ver.stamp = 0;
-	ver.text  = fmt::format("RPCS3 v%s\n%s", rpcs3::version.to_string(), utils::get_system_info());
+    
+	#ifdef RPCS3_GIT_BRANCH
+		ver.text = fmt::format("RPCS3 v%s\nBranch: %s\n%s\n", rpcs3::version.to_string(), RPCS3_GIT_BRANCH, utils::get_system_info());
```

Comment:
I think it can be solved without copying this line.
Maybe add this to rpcs3::version then? Plus don't really need another line for this.

Reply:
Ah yes , I seem to have missed that. I'll shortly change that to one line and put the branch into RPCS3::version.

Reply:
I've attempted to do this in latest but I can't seem to figure out why rpcs3::branch isn't being returned as expected for the log, but it is for gs_frame.

Reply:
There we go. Thanks for the review , I think it should be better now! :)